packages/replayer/src/replayer.ts

```diff
@@ -313,8 +314,8 @@ export const replayEvents: ReplayEventsFn = async (options) => {
     logger.error("Error thrown during replay", error);
 
     if (error instanceof Error && error.name === "MeticulousTimeoutError") {
-      await browser.close();
-      throw error;
+      // If we hit a timeout error it's sometimes because the browser has fully frozen: this means we won't be able to extract coverage logs, screenshot etc.
+      browserHasFrozen = true;
```

Comment:
might be worth checking `browserHasFrozen` is false before taking a screenshot too for safety?